src/System.Runtime.Extensions/src/Resources/Strings.resx

```diff
@@ -364,4 +361,13 @@
   <data name="Arg_SurrogatesNotAllowedAsSingleChar" xml:space="preserve">
     <value>Unicode surrogate characters must be written out as pairs together in the same call, not individually. Consider passing in a character array instead.</value>
   </data>
+  <data name="PlatformNotSupported_AppDomains" xml:space="preserve">
+    <value>Secondary AppDomains are not supported on this platform.</value>
+  </data>
+  <data name="PlatformNotSupported_CAS" xml:space="preserve">
+    <value>Code Access Security is not supported on this platform.</value>
+  </data>
+  <data name="PlatformNotSupported_AppDomain_ResMon" xml:space="preserve">
+    <value>AppDomain Resource Monitoring is not supported on this platform.</value>
```

Comment:
Is "AppDomain Resource Monitoring" a proper name for something?  If not I suggest the 'R' and the 'M' be lowercased.